qiskit_aer/noise/noise_model.py

```diff
@@ -378,6 +379,27 @@ def from_backend(
                     UserWarning,
                 )
             dt = backend.dt
+        elif backend_interface_version <= 1:
+            # BackendV1 will be removed in Qiskit 2.0, so we will remove this soon
```

Comment:
To finalize this notion of "soon" I think maybe we should add both to the comment and to the warning message a concrete version number which drops support for V1 backends (I suggest 0.17)

Reply:
I think it would be beneficial to keep this around for the life cycle of qiskit 1.4. Given that we don't know how many aer releases will happen in this period, maybe it's better for us not to commit to a specific version, it will give us more flexibility to act upon the removal in the future.

Reply:
Agreed, so I think we can merge.